pkg/helm/charts_test.go

```diff
@@ -68,14 +77,9 @@ func TestAddRepos(t *testing.T) {
 	err = c.AddRepos(
 		Repo{Name: "foo", URL: "https://foo.com"},
 		Repo{Name: "foo2", URL: "https://foo2.com"},
-	)
-	assert.NoError(t, err)
-
-	// Only \w characters are allowed in repo names
-	err = c.AddRepos(
 		Repo{Name: "with-dashes", URL: "https://foo.com"},
 	)
-	assert.EqualError(t, err, "1 Repo(s) were skipped. Please check above logs for details")
```

Comment:
This would remove the negative test scenario. Could you please perhaps just add the `with-dashes` line to the positive examples and add for instance a name with ":" as a negative example? 🙂

Reply:
Done!

Reply:
Thank you!